### PR TITLE
Add State Employees Credit Union

### PR DESCRIPTION
**Pull request by gothicman02**
Edited in notepad.  Let me know if there are any issues.

### CODE REVIEW

File: _data/banking.yml [hunk 1]
```diff
@@ -1,4 +1,4 @@
-websites:
+﻿websites:
```
Comment (mxxcon) [inline _data/banking.yml]: Please undo this change.

**Review by mxxcon — CHANGES_REQUESTED** [review]

File: _data/banking.yml (hunk 1 above)
Comment (gothicman02) [reply _data/banking.yml]: I believe this may have happened when I accidentally hit delete on the top line when editting, and corrected it, by undoing.  I did a diff and they look identical currently compared to the original.  Also it appears the checks have passed as well, and I double checked other pull requests to make sure the spacing, line placement, etc. is correct.  Am I missing something?

Comment (mxxcon) [reply _data/banking.yml]: If you edit this file directly on github's website, you'll see there's a character in front of `websites`. I can't tell what exactly it is, but that is what causing this diff. Please delete that char.

Comment (gothicman02) [reply _data/banking.yml]: I see it.  Looks like an extra space, compared to the original.  It has been removed

**Review by Carlgo11 — APPROVED** [review]

**Review by mxxcon — APPROVED** [review]
